exchange_server/metadata.csv.fmt

```diff
@@ -0,0 +1,29 @@
+[metric_name]
```

Comment:
Can we add the linter in a separate PR? Maybe we could add a card for this - it should be possibly something that runs in the CI for all integrations.